docs/crocks/Pred.md

```diff
@@ -0,0 +1,57 @@
+# Pred
+
+`Pred a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Pred` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`empty`](#empty), [`type`](#type) | [`concat`](#concat), [`contramap`](#contramap), [`empty`](#empty),  [`inspect`](#inspect), [`runWith`](#runWith), [`type`](#type), [`value`](#value) |
+
+## Constructor
+
+### empty
+
+`Pred m => () ~> m ()`
+
+### type
+
+`() -> String`
+
+## Instance
+
+### concat
+
+`Pred m => m a ~> a -> m a`
+
+### contramap
+
+`Pred m => m a ~> a -> m ()`
+
+### empty
+
+`Pred m => () ~> m ()`
+
+### inspect
+
+`() => String`
+
+### runWith
+
+`Pred m => m a ~> () -> a`
```

Comment:
So `runWith` literally means *run* this *with* this value. That is the difference between `run` and `runWith`.

This should be:
`Pred m => m a ~> a -> Boolean`